library/sanitize.inc.php

```diff
@@ -177,3 +177,26 @@ function dateEmptySql($sqlColumn, $time = false, $rev = false)
 
     return $stat;
 }
+
+/**
+ * Compares a multibyte unicode string identifier in a case insensitive way to see if the two strings
+ * are semantically identical.  Note that NFKC will treat several 'similar' semantically meaning texts as the same and so
+ * should be used for identifiers (things such as proper nouns, etc).
```

Comment:
hi @adunsulag, should this comment say `should not be used`?

Reply:
I thought about having two separate functions one to be used for general text and one to be used for identifiers, but I decided to just go with the single function.  If you see value in it, I can make the two functions.